fastmri/data/transforms.py

```diff
@@ -509,3 +509,150 @@ def __call__(
             )
 
         return sample
+
+
+class MiniCoilTransform:
+    """
+    Multi-coil compressed transform, for faster prototyping.
+    """
+
+    def __init__(
+        self,
+        mask_func: Optional[MaskFunc] = None,
+        use_seed: Optional[bool] = True,
+        crop_size: Optional[tuple] = None,
+        num_compressed_coils: Optional[int] = None,
+        active_acquisition: Optional[bool] = False,
+    ):
+        """
+        Args:
+            mask_func: Optional; A function that can create a mask of
+                appropriate shape. Defaults to None.
+            use_seed: If True, this class computes a pseudo random number
+                generator seed from the filename. This ensures that the same
+                mask is used for all the slices of a given volume every time.
+            crop_size: Image dimensions for mini MR images.
+            num_compressed_coils: Number of coils to output from coil
+                compression.
+            active_acquisition: Whether the transformation is used for active
+                acquisition. If `True` will also return full kspace.
+                DEPRECATED.
+        """
+        self.mask_func = mask_func
+        self.use_seed = use_seed
+        self.crop_size = crop_size
+        self.num_compressed_coils = num_compressed_coils
+        self.active_acquistion = active_acquisition
+
+    def __call__(self, kspace, mask, target, attrs, fname, slice_num):
+        """
+        Args:
+            kspace: Input k-space of shape (num_coils, rows, cols) for
+                multi-coil data.
+            mask: Mask from the test dataset.
```

Comment:
Looks like this is not used when `self.mask_func` is given. If this is the case, can we add an additional sentence here explaining this?